core/src/martian/swagger.cljc

```diff
@@ -47,23 +47,23 @@
             %) parts)))
 
 (defn- ->handler
-  [{:keys [definitions] :as swagger-map}
+  [{:keys [definitions parameters] :as swagger-map}
    path-item-parameters
    url-pattern
    [method swagger-definition]]
   (when-let [route-name (some-> (:operationId swagger-definition) ->kebab-case-keyword)]
-    (let [path-parts (tokenise-path url-pattern)
+    (let [ref-lookup {:definitions definitions :parameters parameters} path-parts (tokenise-path url-pattern)
```

Comment:
This should be on its own line, and could be `(select-keys swagger-map [:definitions : parameters])`

Reply:
Completed.  Please review latest version.